leasing/importer/usage_distributions.py

```diff
@@ -10,9 +12,9 @@ def __init__(self, stdout=None, stderr=None):
         import oracledb
 
         connection = oracledb.connect(
-            user="mvj",
```

Comment:
Should propably edit git history so that the original pw:s are not readable from github.

Reply:
Even if these are to local test env

Reply:
Those are copypasted from other importer. Those are not working even though